v2/bolt_store.go

```diff
@@ -266,3 +270,67 @@ func (b *BoltStore) GetUint64(key []byte) (uint64, error) {
 func (b *BoltStore) Sync() error {
 	return b.conn.Sync()
 }
+
+// MigratetoV2 reads in the source file path of a BoltDB file 
+// and outputs all the data migrated to a Bbolt destination file
+func MigratetoV2(source, destination string) (*BoltStore, error) {
+	_, err := os.Stat(destination)
```

Comment:
Should we verify this is a BoltDB? How do we do this? 🤔

Reply:
I don't think it matters - if a file is present, whatever type it is, we should err and let the application/operator handle it.